src/lib/team/ui/TeamButton.svelte

```diff
@@ -1,11 +1,15 @@
-<script>
-	//import {toggleMembers} from "../Team.svelte";
-	//import {showMembers}  from "../Team.svelte";
+<script lang="ts">
+	export let showMembers : Boolean;
+	export let onClick: () => Boolean;
+	export let href : string;
```

Comment:
- remove the prop
- modify toggleMembers to programmatically scrollIntoView() when areShown is false